tests/testthat/_snaps/lifecycle.md

```diff
@@ -3,13 +3,13 @@
     Code
       use_lifecycle()
     Message
-      v Adding 'lifecycle' to Imports field in DESCRIPTION
-      * Refer to functions with `lifecycle::fun()`
-      v Adding '@importFrom lifecycle deprecated' to 'R/{TESTPKG}-package.R'
-      v Writing 'NAMESPACE'
-      v Creating 'man/figures/'
-      v Copied SVG badges to 'man/figures/'
-      * Add badges in documentation topics by inserting one of:
+      v Adding lifecycle to 'Imports' field in DESCRIPTION.
+      [ ] Refer to functions with `lifecycle::fun()`.
+      v Adding "@importFrom lifecycle deprecated" to 'R/{TESTPKG}-package.R'.
```

Comment:
Why the mix of double and single quotes?

Reply:
This comes from cli and it's a quoting difference between `{.val VALUE}` and `{.file some/path/to/some/thing}`.

https://github.com/r-lib/cli/blob/1b5e691a44497c8a3708ed33ab2026c640a2c66c/R/format.R#L73

https://github.com/r-lib/cli/blob/1b5e691a44497c8a3708ed33ab2026c640a2c66c/R/themes.R#L365

https://github.com/r-lib/cli/blob/1b5e691a44497c8a3708ed33ab2026c640a2c66c/R/themes.R#L294-L300